pwiz_tools/Skyline/Model/AlphaPeptDeep/AlphapeptdeepLibraryBuilder.cs

```diff
@@ -1,4 +1,23 @@
-﻿using System;
+﻿/*
+ * Maintainer: David Shteynberg <david.shteynberg .at. proton.me>,
+ *                  MacCoss Lab, Department of Genome Sciences, UW
```

Comment:
"Maintainer" isn't something you'll find anywhere in the rest of the code. Just use "Author", which is Skyline-speak for "owner" or "knows more about it than anyone else, presumably". And, use your proteinms email.